src/algebra/lie/basic.lean

```diff
@@ -774,7 +781,9 @@ instance : partial_order (lie_submodule R L M) :=
 { le := λ N N', ∀ ⦃x⦄, x ∈ N → x ∈ N', -- Overriding `le` like this gives a better defeq.
   ..partial_order.lift (coe : lie_submodule R L M → set M) coe_injective }
 
-lemma le_def (N N' : lie_submodule R L M) : N ≤ N' ↔ (N : set M) ⊆ N' := iff.rfl
+lemma le_def : N ≤ N' ↔ (N : set M) ⊆ N' := iff.rfl
+
+lemma le_def_submodule : N ≤ N' ↔ (N : submodule R M) ≤ N' := iff.rfl
```

Comment:
If you swap the two sides (and call it something like `coe_le_coe_submodule`, then it would make a useful simp-lemma.